pkg/scheduler/plugins/elasticquota/core/quota_info.go

```diff
@@ -149,10 +149,6 @@ func (qi *QuotaInfo) updateQuotaInfoFromRemote(quotaInfo *QuotaInfo) {
 	qi.lock.Lock()
 	defer qi.lock.Unlock()
 
-	if quotaInfo.Name == extension.SystemQuotaName {
```

Comment:
why?

Reply:
As we discussed before, systemQuotaGroup will be created in memory without sharedWeight, but the systemQoutaGroup's CRD will be filled with sharedWeight, the webhook never checks sharedWeight, so it will always rebuild the tree when updateSystemQuotaGroup, but the tree is no need to rebuild indeed.

Reply:
ok